src/components/TextInput/styles.tsx

```diff
@@ -1,15 +1,15 @@
 import styled from 'styled-components';
-import { Inter } from 'next/font/google';
-import COLORS from '../../styles/colors';
+import { Open_Sans as OpenSans } from 'next/font/google';
+import COLORS from '@/styles/colors';
 
-const inter = Inter({ subsets: ['latin'] });
+const openSans = OpenSans({ subsets: ['latin'] });
 
 export const InputText = styled.input<{ $error: boolean }>`
+  // is possible to inherit text styles from H4?
```

Comment:
yee it is! you can make this InputText = styled(H4) and then in the actual jsx portion you can pass in the prop "as='input'" to make sure that the component is still being rendered as an input component

Reply:
gotcha, thank you! this is super helpful, suraj!